distribution/src/resources/drill-override-example.conf

```diff
@@ -93,6 +93,13 @@ drill.exec: {
       credentials: true
     }
   },
+  # Below SSL parameters need to be set for custom transport layer settings.
+  ssl{
+    keyStore: "/keystore.file",
```

Comment:
The names in this example file are not consistent with the names uses in ExecConstants. keyStore and trustStore instead of keyStorePath, trustStorePath.